tokens/color/purple.yaml

```diff
@@ -0,0 +1,10 @@
+color:
```

Comment:
Is there any reason that each color range having their own file could hurt us in the future if we wanted to, say, change all purples to blue?

Reply:
Or would it be that the value of the token concepts allows you to change the token we're calling in vs. changing the token itself?

Reply:
in the end it all compiles down to one flat list. style-dictionary will warn us about name conflicts

Reply:
so, if in `blue.yml` we have
```yml
color:
  blue:
    yadda:
      value: 'ya'
```
and in `purple.yml` we have
```yml
color:
  purple:
    badda:
      value: 'bind'
```
then we'll end up with
```css
--rh-color-blue-yadda: ya;
--rh-color-purple-badda: bing;
```